components/Pagination/Pagination.js

```diff
@@ -10,19 +10,40 @@ Pagination.propTypes = {
   className: string,
   currentPage: number.isRequired,
   totalPages: number.isRequired,
+  maxElements: number,
 };
 
 Pagination.defaultProps = {
   className: undefined,
+  maxElements: 11,
 };
 
 // eslint-disable-next-line react/prop-types
-const PaginationItems = ({ currentPage, totalPages }) => {
-  const shouldTruncateStart = currentPage - 1 > 5;
-  const shouldTruncateEnd = totalPages - currentPage > 5;
+const PaginationItems = ({ currentPage, totalPages, maxLength }) => {
+  // maxElements is maximum length of the Pagination Bar, should be an odd integer, delault is 11
+  const maxElements = maxLength % 2 === 0 ? maxLength - 1 : maxLength;
```

Comment:
Lets drop `maxElements` since we don't need that aspect to be customizeable.

Reply:
Okay :+1: